pks-pipelines/minimum-release-verification/utils/lock-to-env.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eux pipefail
```

Comment:
I thought that you needed an `-o` in order to use `pipefail`

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/